gnd/src/test/java/com/google/android/gnd/persistence/local/LocalDataStoreTest.java

```diff
@@ -167,137 +207,97 @@ public void setUp() {
+    Project project = TEST_PROJECT;
+    localDataStore.insertOrUpdateProject(project).subscribe();
     localDataStore.getProjectById(project.getId()).test().assertValue(project);
   }
 
   @Test
   public void testDeleteProject() {
-    Project project = createTestProject();
-    localDataStore.insertOrUpdateProject(project).test().assertComplete();
+    Project project = TEST_PROJECT;
+    localDataStore.insertOrUpdateProject(project).subscribe();
     localDataStore.deleteProject(project).test().assertComplete();
     localDataStore.getProjects().test().assertValue(AbstractCollection::isEmpty);
   }
 
   @Test
   public void testGetUser() {
-    User user = createTestUser();
-    localDataStore.insertOrUpdateUser(user).test().assertComplete();
+    User user = TEST_USER;
+    localDataStore.insertOrUpdateUser(user).subscribe();
     localDataStore.getUser(user.getId()).test().assertValue(user);
   }
 
   @Test
   public void testApplyAndEnqueue_featureMutation() {
-    User user = createTestUser();
-    localDataStore.insertOrUpdateUser(user).test().assertComplete();
-
-    Project project = createTestProject();
-    localDataStore.insertOrUpdateProject(project).test().assertComplete();
+    localDataStore.insertOrUpdateUser(TEST_USER).subscribe();
+    localDataStore.insertOrUpdateProject(TEST_PROJECT).subscribe();
 
-    Layer layer = project.getLayers().get(0);
-    FeatureMutation mutation =
-        createTestFeatureMutation(user.getId(), project.getId(), layer.getId());
+    FeatureMutation mutation = TEST_FEATURE_MUTATION;
     localDataStore.applyAndEnqueue(mutation).test().assertComplete();
 
-    ImmutableList<Mutation> savedMutations =
-        localDataStore.getPendingMutations(mutation.getFeatureId()).blockingGet();
-    Assert.assertEquals(1, savedMutations.size());
-
     // assert that mutation is saved to local database
-    FeatureMutation savedMutation = (FeatureMutation) savedMutations.get(0);
-    Assert.assertEquals(mutation.getNewLocation(), savedMutation.getNewLocation());
-    Assert.assertEquals(mutation.getType(), savedMutation.getType());
-    Assert.assertEquals(mutation.getUserId(), savedMutation.getUserId());
-    Assert.assertEquals(mutation.getProjectId(), savedMutation.getProjectId());
-    Assert.assertEquals(mutation.getFeatureId(), savedMutation.getFeatureId());
-    Assert.assertEquals(mutation.getLayerId(), savedMutation.getLayerId());
-    Assert.assertEquals(mutation.getClientTimestamp(), savedMutation.getClientTimestamp());
-    Assert.assertEquals(0, savedMutation.getRetryCount());
-    Assert.assertNull(savedMutation.getLastError());
+    localDataStore
+        .getPendingMutations(mutation.getFeatureId())
+        .test()
+        .assertValue(ImmutableList.of(mutation));
 
     // assert feature is saved to local database
-    Feature feature = localDataStore.getFeature(project, mutation.getFeatureId()).blockingGet();
+    Feature feature =
```

Comment:
Can this entire block be replaced with:
```
   localDataStore.getFeature(TEST_PROJECT, mutation.getFeatureId()).test().assertValue(TEST_FEATURE);
```
?

Reply:
That would require mocking a static method. For that power mock can be used. https://github.com/robolectric/robolectric/wiki/Using-PowerMock

But I didn't have any success with that.